server/test/repository/ApplicationStatusesRepositoryTest.java

```diff
@@ -85,4 +89,31 @@ public void canQueryForOboseleteApplicationStatuses() {
     assertThat(statusDefinitionsResults.get(0).getStatuses().get(0).statusText())
         .isEqualTo("Approved");
   }
+
+  @Test
+  public void canUpdateApplicationStatuses() {
+    Long uniqueProgramId = new Random().nextLong();
+    ProgramModel program =
+        ProgramBuilder.newActiveProgram("Updateprogram" + uniqueProgramId, "description").build();
+    String programName = program.getProgramDefinition().adminName();
+    StatusDefinitions statusDefinitions = new StatusDefinitions(ImmutableList.of(APPROVED_STATUS));
+    ApplicationStatusesModel applicationStatusesModel =
+        new ApplicationStatusesModel(
+            programName, statusDefinitions, StatusDefinitionsLifecycleStage.ACTIVE);
+    applicationStatusesModel.save();
+    StatusDefinitions statusDefinitionsResult = repo.lookupActiveStatusDefinitions(programName);
+    assertStatusDefinitions(statusDefinitionsResult, "Approved");
```

Comment:
**nit, non-blocking**: I might add a line between lines 103 and 104 to make it clear this is a "pre-assert" step that's a little distinct from the "arrange" step before it.